db/migrate/20160119125950_add_created_on_for_container_entities.rb

```diff
@@ -9,7 +9,7 @@ def change
       rename_column t, :creation_timestamp, :ems_created_on
 
       say_with_time("adding created_on datetime to all existing #{t.to_s.tr("_", " ")}") do
-        t.to_s.classify.constantize.update_all("created_on=ems_created_on")
+        t.to_s.classify.constantize.unscoped.update_all("created_on=ems_created_on")
```

Comment:
Its obviously bad to change a past migration but I think this change is innocent enough.
without this change, running ```rake evm:db:reset``` will encounter an error having to do with ```deleted_on ``` not being an existing field yet while trying to query it in the default scope.